src/components/expandable-text/expandable-text.tsx

```diff
@@ -49,18 +49,15 @@ const ExpandableText: new (props?: any, context?: any) => any = withText({
     }
   }, [isFinalized]);
 
-  useEffect(() => {
-    if (props.onExpand) {
-      props.onExpand(isTextExpanded);
-    }
-  }, [isTextExpanded]);
-
   const onClick = (e: MouseEvent) => {
     if (props.onClick) {
       props.onClick(e);
     }
 
     setIsTextExpanded(!isTextExpanded);
+    if (props.onExpand) {
```

Comment:
you can extract the duplicated code to a function

Reply:
fixed